<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationAllocationDialog</class>
 <widget class="QDialog" name="LocationAllocationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="combo_candidates"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_candidates">
       <property name="text">
        <string>Candidate locations</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_facilities"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_facilities">
       <property name="text">
        <string>Number of facilities</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_deadline"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_end"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_deadline">
       <property name="text">
        <string>Deadline</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="combo_demands"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_demands">
       <property name="text">
        <string>Demands</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_end">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_start"/>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_useGrid">
       <property name="text">
        <string>Use grid</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_cellSize">
       <property name="text">
        <string>Cell size</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit_cellSize"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocationAllocationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocationAllocationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
